backend/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.2.4</version>
+	</parent>
+
+
+	<groupId>ca.gov.dts-stn</groupId>
+	<artifactId>cdcp-api</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
```

Comment:
Consider using `<version>${revision}</version>`, along with a `<revision>0.0.0-00000000-0000</revision>` property so we can set the version at build time.

Reply:
[Added `revision` property](https://github.com/DTS-STN/canadian-dental-care-plan/compare/e4b6b4e5d79d4f716e14946054039a3e1e8e14d9..57c82567f8066b3daa2195f0ca7ad35ecbf1b414#diff-34049c3bc6deee4bbf269544338e0450399140da63fec684096d1ae0ce70b4bb)